### PR TITLE
docs: adding recent faqs

### PR DESCRIPTION
**Pull request by RobPasMue**
## Description
Many FAQs have come along recently on how to build Docker containers, install the Ansys Geometry service and so on. This solves these requests

## Issue linked
#1022 for example

## Checklist
- [X] I have tested my changes locally.
- [X] I have added necessary documentation or updated existing documentation.
- [X] I have followed the coding style guidelines of this project.
- [X] I have added appropriate unit tests.
- [X] I have reviewed my changes before submitting this pull request.
- [X] I have linked the issue or issues that are solved to the PR if any.
- [X] I have assigned this PR to myself.
- [X] I have made sure that the title of my PR follows [Conventional commits style](https://www.conventionalcommits.org/en/v1.0.0/#summary) (e.g. ``feat: extrude circle to cylinder``)

### CODE REVIEW

**Comment by RobPasMue** [comment]
@PipKat - as always, I'd appreciate your review here =)

**Review by MaxJPRey — COMMENTED** [review]
This is an excellent idea.
We should generalize it to many other projects.

File: doc/source/getting_started/faq.rst [hunk 1]
```diff
@@ -12,6 +12,39 @@ Design Language (APDL), Ansys Fluent, and other Ansys products.
...
+
+Overall, the command to install the Ansys Geometry service with the unified installer is:
+
+.. code-block:: bash
+
+   setup.exe -silent -geometryservice
+
+You can verify that the installation was successful by checking whether the
+product has been installed on your file directory. If you are using the default
+installation directory, the product will be installed in the following directory:
```
Comment (RobPasMue) [inline doc/source/getting_started/faq.rst:40]:
```suggestion
installation directory, the product is installed in the following directory:
```

File: doc/source/getting_started/faq.rst [hunk 2]
```diff
@@ -34,6 +67,18 @@ such as the Ansys Mechanical Enterprise license, that also allow users to run
 these Ansys products. However, the Geometry service is only compatible with
 the **Discovery Modeling** license.
 
+How can I build my own Docker image for the Ansys Geometry service?
+-------------------------------------------------------------------
```
Comment (RobPasMue) [inline doc/source/getting_started/faq.rst:71]:
```suggestion
How to build the Docker image for the Ansys Geometry service?
-------------------------------------------------------------
```

File: doc/source/getting_started/faq.rst [hunk 3]
```diff
@@ -34,6 +67,18 @@ such as the Ansys Mechanical Enterprise license, that also allow users to run
...
+
+.. note::
+
+   This question is answered in https://github.com/ansys/pyansys-geometry/discussions/883
+
+To build your own Docker image for the Ansys Geometry service, users should follow
+the instructions provided in :ref:`ref_build_windows_docker_image_from_ansys_installation`. The
+resulting image will be a Windows-based Docker image that contains the Ansys Geometry
```
Comment (RobPasMue) [inline doc/source/getting_started/faq.rst:79]:
```suggestion
resulting image is a Windows-based Docker image that contains the Ansys Geometry
```

**Pull request merged**